src/module/actor/character/document.ts

```diff
@@ -2109,6 +1984,9 @@ interface CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocument
     flags: CharacterFlags;
     readonly _source: CharacterSource;
     system: CharacterSystemData;
+
+    /** Skills for the character, built during data prep */
+    skills: CharacterSkills;
```

Comment:
Send 'er up top

Reply:
I thought we wanted to put the ones that are extended down there since I saw something recently added in the actor base case. Sure, I'll move everything on up.